src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -197,13 +257,10 @@ public void RequestAnalysis(string path, string charset, IIssueConsumer consumer
             request.File.Add(new InputFile
             {
                 Path = path,
-                Charset = charset,
+                Charset = charset                
```

Comment:
Let's remove this trailing spaces (you can add the `Trailing Whitespace Visualizer` extension into your VS).